trade-finance-manager-ui/component-tests/utilisation-reports/add-to-an-existing-payment.component-test.ts

```diff
@@ -73,13 +74,13 @@ describe(page, () => {
     wrapper.expectText(tableSelector).toContain('JPY 1');
   });
 
-  it('should render available payment groups radio input', () => {
+  it('should render available payment groups', () => {
     // Arrange
     const addToAnExistingPaymentViewModel = anAddToAnExistingPaymentViewModel();
     const wrapper = render(addToAnExistingPaymentViewModel);
 
     // Assert
-    wrapper.expectElement('div[data-cy="payment-group-radio-input"]').toExist();
+    wrapper.expectElement('div[data-cy="available-payment-groups"]').toExist();
```

Comment:
I think the present name with the 'available' reference is suitable as we want consistent naming across the codebase. In central we're filtering payment groups to get those we can add fee records to (the 'available payment groups'), and I think it makes sense that this naming makes it way through the different services. This would be a really big refactor if we wanted to rename this across the services and I think this would fall out of scope of this PR.

Reply:
Understood @sw-mattw and I agree RE out of scope for this PR.

That seems to make sense to me in the central API, but for the UI consumption I think just "payment groups" would be cleaner, as AFAIK the UI is (or should?) simply consume a set of payment groups from the API.

Having "available" in the naming convention says to me that you could potentially filter by other things e.g "unavailable", "closed", "dynamic" and things like that

Reply:
I've removed 'available' references in the view models and templates, but kept this phrasing in the controllers where we make calls to TFM/central APIs and they return objects with this naming. Is that suitable? Thanks